packages/e2e-tests/helpers/commands.ts

```diff
@@ -12,10 +12,21 @@ export async function quickOpen(page: Page, url: string): Promise<void> {
   await debugPrint(page, `Filtering files by "${chalk.bold(url)}"`, "quickOpen");
   await page.keyboard.type(url);
 
+  await waitFor(async () => {
+    const loader = page.getByText("Loading results...");
+    const isVisible = await loader.isVisible();
+    expect(isVisible).toBe(false);
+  });
+
   await debugPrint(page, `Opening file "${chalk.bold(url)}"`, "quickOpen");
   const sourceRow = await page.waitForSelector(
-    `[data-test-name="QuickOpenResultsList-Row"]:has-text("${url}")`
+    `[data-test-name="QuickOpenResultsList-Row"].selected:has-text("${url}")`
   );
+  await waitFor(async () => {
+    const isVisible = await sourceRow.isVisible();
+    expect(isVisible).toBe(true);
+  });
```

Comment:
I actually thought `waitForSelector` would do _the same thing_ but... I guess not?

Reply:
Oh, that's what we already _had_. Cool, that simplifies things I guess.